src/components/daily-checkin/daily-checkin.component.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./daily-checkin.styles.scss";
+
+const DailyCheckin = () => (
+  <div className="daily-checkin">
+    <div className="inline-daily-checkin">
+      <div> Daily Checkin</div>
```

Comment:
why is there a space?

Reply:
Hey Andy, thanks for reviewing this, this happens to be my code style because I find it easier to read when there is one space between divs